index2.html

```diff
@@ -0,0 +1,230 @@
+            <button class="resume">Get my resume</button>
+          </div>
+          <hr class="about-breaker">
+          <div class="skills">
+            <ul class="tools-list">
+                <ul class="listtopography">
+                 <li class="header-list" id="language">
+                  <img id="titles" src="img/Rectangle 55.png" alt="Languages">
+                  <h1>Language</h1>
+                 </li>
+                 <ul class="mini-list">
+                  <li class="detail">JavaScript</li>
+                  <li class="detail">Ruby</li>
+                  <li class="detail">HTML</li>
+                  <li class="detail">CSS</li>
+                 </ul>
+                </ul>
+                <ul class="listtopography">
+                  <li class="header-list" id="frameworks">
+                    <img id="titles" src="img/Rectangle 56.png" alt="Languages">
+                    <h1>Frameworks</h1>
+                  </li>
+                  <ul class="mini-list">
+                    <li class="detail">React.js</li>
+                    <li class="detail">Ruby on rails</li>
+                    <li class="detail">RSpec</li>
+                    <li class="detail">Capybara</li>
+                  </ul>
+                 </ul>
+                 <ul class="listtopography">
+                    <li class="header-list" id="skills">
+                      <img id="titles" src="img/Rectangle 57.png" alt="Languages">
+                      <h1>Skills</h1>
+                     </li>
+                     <ul class="mini-list">
+                      <li class="detail">Database Management</li>
+                      <li class="detail">Version Control</li>
+                      <li class="detail">CLI</li>
+                      <li class="detail">API Design</li>
+                      <li class="detail">Web Development</li>
+                     </ul>
+                  </ul>
+              </ul>
+          </div>
+       </section>
+       <section class="bottom-footer">
+           <div class="contact-section">
+              <p class="contact-text"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+           </div>
+           <form action="https://formspree.io/html" method="post">
+            <fieldset> 
+            <label> 
+           <input id="Full-name" name="" type="text" required placeholder="Full name" max-length="30" />
+            </label>
+            <label>
+           <input id="email" name="" type="email" required placeholder="Email address" />
+          </label>
+          </fieldset>
+            <textarea name="" id="mess" placeholder="Write me something..." cols="50" rows="10" maxlength="500"></textarea>
+         <button class="Submit-btn">
+           Get in touch
+         </button>
+        </form>
+        <hr class="form-breaker">
```

Comment:
- [ ] The previous reviewer also mentioned that the `horizontal line` needs to extend across the entire width of the page as seen in the original design :wink: 
- Your Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/c5698278-2be8-47ed-a82e-7c6dde2d0fae)

- The Figma Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/a93ffd5b-0fac-4cbe-8e4d-5ee2356f35e8)